src/test/java/gov/healthit/chpl/aqa/stepDefinitions/Base.java

```diff
@@ -95,6 +96,37 @@ public void checkIfFileIsDownloaded(final String fileName)throws FileNotFoundExc
         }
     }
 
+    /**
+     * Checks if the recent downloaded file is downloaded completely.
+     * file name starts with name of the file and ends with extension.
+     * @param fileName is the name of file
+     * @param extension of the file
+     * @throws FileNotFoundException if the expected file not found
+     */
+    public void checkCompleteFileDownload(final String fileName, final String extension) throws FileNotFoundException {
+        boolean foundFile = false;
+        int retryCount = 0;
+        try {
+            while (!foundFile && retryCount <= MAX_RETRYCOUNT) {
+                File[] files = Hooks.getDownloadDirectory().listFiles();
+                for (int i = 0; i < files.length; i++) {
+                    Long currentTime = System.currentTimeMillis();
+                    if ((files[i].getName().startsWith(fileName) && files[i].getName().endsWith(extension)) && ((currentTime - files[i].lastModified()) < SLEEP_TIME * COUNT)) {
+                        foundFile = true;
+                        break;
+                    }
+                }
+                retryCount++;
+                Thread.sleep(SLEEP_TIME);
```

Comment:
Probably should surround this with an "if !foundFile" so if you do find it we don't needlessly wait another 5 seconds